data/campaigns/Under_the_Burning_Suns/units/units.cfg

```diff
@@ -16,7 +16,7 @@
     male_name= _ "Unknown"
     female_name= _ "Unknown"
     plural_name= _ "Unknown"
-    description="<img>src=units/unknown-unit.png~RC(magenta>red) align=left float=yes</img>" + _ "
+    description="<img>src=units/unknown-unit.png~RC(magenta>red) "+ _ "align=left float=yes"+"</img>"
```

Comment:
This is utterly crazy and horribly wrong. The existing code is correct, so you should probably just revert to that.

Reply:
Revert.